### ISSUE TITLE
Burrow service frequent down 

### ISSUE THREAD

**Issue by BabuVelu** [linkedin/Burrow#365]
Team, 

Need your help to fix the Burrow service frequent down issue.

Error Snip

<Snip>

panic: interface conversion: interface {} is nil, not *storage.brokerOffset

goroutine 85 [running]:
github.com/linkedin/Burrow/core/internal/storage.(*InMemoryStorage).fetchTopic(0xc4201d65a0, 0xc42100b8f0, 0xc42034d800)
       /opt/work/src/github.com/linkedin/Burrow/core/internal/storage/inmemory.go:611 +0x3c8
github.com/linkedin/Burrow/core/internal/storage.(*InMemoryStorage).(github.com/linkedin/Burrow/core/internal/storage.fetchTopic)-fm(0xc42100b8f0, 0xc42034d800)
       /opt/work/src/github.com/linkedin/Burrow/core/internal/storage/inmemory.go:182 +0x3e
github.com/linkedin/Burrow/core/internal/storage.(*InMemoryStorage).requestWorker(0xc4201d65a0, 0x11, 0xc4201d4900)
       /opt/work/src/github.com/linkedin/Burrow/core/internal/storage/inmemory.go:190 +0x105b
created by github.com/linkedin/Burrow/core/internal/storage.(*InMemoryStorage).Start
       /opt/work/src/github.com/linkedin/Burrow/core/internal/storage/inmemory.go:144 +0x28b
</Snip>

**Comment by luiscarbajo23** [comment]
Same here!!

goroutine 24 [running]:
github.com/linkedin/Burrow/core/internal/storage.(*InMemoryStorage).fetchTopic(0xc420085cc0, 0xc4205adc70, 0xc4206f6ba0)
	/go/src/github.com/linkedin/Burrow/core/internal/storage/inmemory.go:611 +0x3ee
github.com/linkedin/Burrow/core/internal/storage.(*InMemoryStorage).(github.com/linkedin/Burrow/core/internal/storage.fetchTopic)-fm(0xc4205adc70, 0xc4206f6ba0)
	/go/src/github.com/linkedin/Burrow/core/internal/storage/inmemory.go:182 +0x3e
github.com/linkedin/Burrow/core/internal/storage.(*InMemoryStorage).requestWorker(0xc420085cc0, 0x3, 0xc4200602a0)
	/go/src/github.com/linkedin/Burrow/core/internal/storage/inmemory.go:190 +0x1003
created by github.com/linkedin/Burrow/core/internal/storage.(*InMemoryStorage).Start
	/go/src/github.com/linkedin/Burrow/core/internal/storage/inmemory.go:144 +0x336

Thank you!!! 👍

**Comment by fvigotti** [comment]
same here : 
```
panic: interface conversion: interface {} is nil, not *storage.brokerOffset

goroutine 51 [running]:
github.com/linkedin/Burrow/core/internal/storage.(*InMemoryStorage).fetchTopic(0xc420236000, 0xc420345500, 0xc4203281e0)
        /go/src/github.com/linkedin/Burrow/core/internal/storage/inmemory.go:611 +0x3ee
github.com/linkedin/Burrow/core/internal/storage.(*InMemoryStorage).(github.com/linkedin/Burrow/core/internal/storage.fetchTopic)-fm(0xc420345500, 0xc4203281e0)
        /go/src/github.com/linkedin/Burrow/core/internal/storage/inmemory.go:182 +0x3e
github.com/linkedin/Burrow/core/internal/storage.(*InMemoryStorage).requestWorker(0xc420236000, 0x10, 0xc42023e780)
        /go/src/github.com/linkedin/Burrow/core/internal/storage/inmemory.go:190 +0x1003
created by github.com/linkedin/Burrow/core/internal/storage.(*InMemoryStorage).Start
        /go/src/github.com/linkedin/Burrow/core/internal/storage/inmemory.go:144 +0x336
```

**Comment by toddpalino** [comment]
Merged #502
